apps/notifications/src/panel/templates/block-user.jsx

```diff
@@ -162,15 +162,28 @@ export class UserBlock extends Component {
 			);
 		}
 
+		const userId = this.props.block?.meta?.ids?.user;
+		const readerProfileUrl = userId ? `https://wordpress.com/reader/users/${ userId }` : home_url;
```

Comment:
Can we make these relative URLs rather than absolute? That way we can easily stay in the current environment.

Reply:
The problem with that is this is also bundled and deployed to the standalone notifications app build. So when opening the notifications panel on `yoursite.com/wp-admin` or on a p2, it needs the full path to get to the right place.

I don't think staying in the environment is too impactful here. This seems like it would be a fairly uncommon flow for calypso environment specific testing, as its a bit of an environment agnostic feature the way its bundled and deployed. Its also setup to open in a separate tab which somewhat helps there too.

Reply:
`https://wordpress.com/reader/users/${ userId }` works in 90% of cases except for one very high-profile case. Matt's user ID is 5, and the username 5 is taken.

Reply:
Wow nice catch. 

this seems even broader issue in general - how can we let the route support both id and username, if people can create numeric usernames that conflict with id?